tests/ComposerJsonReaderTest.php

```diff
@@ -301,6 +301,13 @@ public static function provideGetPhpCases(): iterable
             '8.1',
             '{
     "require": { "php": "  8.1.*  ||  8.2.*  ||  8.3.*  ||  8.4.*  " }
+}',
+        ];
+
+        yield 'version with >= and a space' => [
```

Comment:
can you put that also next to this?
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/9203/files#diff-5a1bf715057efb2f27a38a872e81c79910b78dd6338e5c3e4f595a8cce83dc6fR116